tools/test_runner.js

```diff
@@ -18,6 +18,8 @@ var console_wrapper = require('common_js/module/console');
 var builtin_modules =
   Object.keys(process.native_sources).concat(Object.keys(process.binding));
 
+var default_timeout = 240; // (in seconds)
```

Comment:
@bzsolt Would it be possible to have it as a command line parameter? 240 is fine for me, but if not I could override it.